terragrunt/aws/cloud_asset_inventory/lambda.tf

```diff
@@ -0,0 +1,77 @@
+#
+# Lambda: zip
+#
+data "archive_file" "neo4j_to_sentinel" {
+  type        = "zip"
+  source_dir  = "src/sentinel_ingestor/dist"
+  output_path = "/tmp/neo4j_to_sentinel.py.zip"
+
+  depends_on = [
+    null_resource.lambda_build
+  ]
+}
+
+resource "aws_lambda_function" "neo4j_to_sentinel" {
+  filename      = data.archive_file.neo4j_to_sentinel.output_path
+  function_name = "sentinel-neo4j-forwarder"
+  handler       = "neo4j_to_sentinel.handler"
+  runtime       = "python3.9"
+  timeout       = 300
+  memory_size   = 256
+  role          = aws_iam_role.neo4j_to_sentinel_lambda.arn
+
+  source_code_hash = data.archive_file.neo4j_to_sentinel.output_base64sha256
+
+  vpc_config {
+    subnet_ids         = var.vpc_private_subnet_ids
+    security_group_ids = [aws_security_group.cartography.id]
+  }
+
+  environment {
+    variables = {
+      CUSTOMER_ID            = var.customer_id
+      LOG_TYPE               = "CartographyTest"
+      SHARED_KEY             = var.shared_key
+      NEO4J_URI              = "bolt://neo4j.internal.local:7687"
+      NEO4J_USER             = "neo4j"
+      NEO4J_SECRETS_PASSWORD = aws_ssm_parameter.neo4j_password.value
+    }
+  }
+
+  tracing_config {
+    mode = "Active"
+  }
+
+  layers = ["arn:aws:lambda:ca-central-1:283582579564:layer:aws-sentinel-connector-layer:6"]
```

Comment:
I wish they'd make this easier for Docker image based Lambdas.

Reply:
@patheard it's not as clean but i found [this](https://github.com/cds-snc/notification-api/blob/main/bin/get_newrelic_layer.sh) + [unzip in dockerfile](https://github.com/cds-snc/notification-api/blob/main/ci/Dockerfile.lambda#L36-L37) to be a good alternative thats low effort. But it'll never win against a one liner 😂

Reply:
💯